contracts/src/v0.8/ccip/test/pools/GHO/GhoTokenPoolEthereumBridgeLimit.t.sol

```diff
@@ -0,0 +1,1043 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {GhoToken} from "@aave-gho-core/gho/GhoToken.sol";
+
+import {IPoolV1} from "../../../interfaces/IPool.sol";
+import {Pool} from "../../../libraries/Pool.sol";
+import {UpgradeableLockReleaseTokenPool} from "../../../pools/GHO/UpgradeableLockReleaseTokenPool.sol";
+import {GhoBaseTest} from "./GhoBaseTest.t.sol";
+
+contract GhoTokenPoolEthereumBridgeLimitSetup is GhoBaseTest {
+  error BridgeLimitExceeded(uint256 bridgeLimit);
+
+  UtilsStorage public s;
+
+  function setUp() public virtual override {
+    // Ethereum with id 0
+    s.chainsList.push(0);
+    s.tokens[0] = address(new GhoToken(AAVE_DAO));
+    s.pools[0] = _deployUpgradeableLockReleaseTokenPool(
+      s.tokens[0],
+      RMN_PROXY,
+      ROUTER,
+      AAVE_DAO,
+      INITIAL_BRIDGE_LIMIT,
+      PROXY_ADMIN
+    );
+
+    // Mock calls for bridging
+    vm.mockCall(ROUTER, abi.encodeWithSelector(bytes4(keccak256("getOnRamp(uint64)"))), abi.encode(RAMP));
+    vm.mockCall(ROUTER, abi.encodeWithSelector(bytes4(keccak256("isOffRamp(uint64,address)"))), abi.encode(true));
+    vm.mockCall(RMN_PROXY, abi.encodeWithSelector(bytes4(keccak256("isCursed()"))), abi.encode(false));
+    vm.mockCall(RMN_PROXY, abi.encodeWithSelector(bytes4(keccak256("isCursed(bytes16)"))), abi.encode(false));
```

Comment:
it seems the last one no longer exists, right?

Reply:
yes, correct! [ae55d15](https://github.com/aave/ccip/pull/20/commits/ae55d158419f56b0dfa9b7f833043c5d33391eae)